### ISSUE TITLE
Training loss curve

### ISSUE THREAD

**Issue by hukkai** [magic-research/PLLaVA#66]
Dear authors, thanks for your insightful work! I was able to reproduce your model and my trained video-LLM can have meaningful outputs. However, since it is my first time to train a video-LLM model, I am not sure if I did it correctly since my loss did not decrease too much. Is it possible for you to share one of your training log or can you check if my training loss curve looks correct?

![Figure_1](https://github.com/magic-research/PLLaVA/assets/35267818/6fcf6c6c-2a84-4e19-9601-0cd775a0fba9)
This is my training loss curve of PLLaVA-7B using the default setting (averaged in a sliding window of 1024)

**Comment by Namzakku** [comment]
@hukkai Hi, I also got into same problem. Have you found any solution?
Btw, what is your setting for batch_size and learning rate?

**Comment by hukkai** [comment]
@Namzakku I am using batch size = 4 and did not change anything else.
